src/vm/marshalnative.cpp

```diff
@@ -54,7 +54,7 @@
 //
 // NumParamBytes
 // Counts # of parameter bytes
-INT32 QCALLTYPE MarshalNative::NumParamBytes(MethodDesc * pMD)
+INT32 QCALLTYPE MarshalNative::NumParamBytes(MethodDesc * pMD, CLR_BOOL fForStdCallDelegate)
```

Comment:
`CLR_BOOL` is not correct here.

Reply:
Good catch, will fix.

I do see `QCALLTYPE` and `CLR_BOOL` used together in a few places, such as https://github.com/dotnet/coreclr/blob/fae8cc77819402d143042da5cffeb36dd2c0aecf/src/vm/comthreadpool.h#L38. Is it worth opening an issue for these or should we just let existing code slide?